src/components/CurrentPlan/CurrentPlan.js

```diff
@@ -93,10 +93,10 @@ class CurrentPlan extends PureComponent {
                   currency = subItem.nextPaymentCurrency;
                   description =
                     subItem.status === 'active'
-                      ? `Next payment is on ${dateFormat(
+                      ? `${t('Next payment is on')} ${dateFormat(
```

Comment:
I think that in such cases we should add a variable as a part of translations. In other languages, the date can be placed not at the end